x-pack/plugins/alerting/server/lib/process_alerts.ts

```diff
@@ -152,6 +190,11 @@ function processAlertsLimitReached<
         ...(state.start ? { start: state.start } : {}),
         ...(duration !== undefined ? { duration } : {}),
       });
+
+      // this alert is still active
+      if (setFlapping) {
+        activeAlerts[id].updateFlappingHistory(false);
```

Comment:
I am wondering how we should handle updating flapping when the process alerts limit is reached, I just set it to false but I am not sure if that is what you would expect.

Reply:
Resolved in this commit c137d4d0674cf656c3c61d2c95bf7eec6219c2d9